### DISCUSSION TITLE
How to use/install older releases of Spotx?

### DISCUSSION THREAD

**Discussion by NeuroCPP** [SpotX-Official/SpotX#162] [Question]
Yes, I have read the FAQs 
now as the question suggests that I want to use older version of patch for eg. [v1.4](https://github.com/amd64fox/SpotX/tree/1.4)?

**Comment by amd64fox** [comment]
![91](https://user-images.githubusercontent.com/62529699/191972887-78ef0c82-7aa5-4ac9-b770-8277935ee5ab.gif)

**Reply by NeuroCPP** [reply]
Thanks!

**Reply by NeuroCPP** [reply]
I was also wondering at first that might be by downloading the source code of v1.4
didn't do it though 😅
